contracts/staking/SimpleStaking.sol

```diff
@@ -90,9 +90,6 @@ contract SimpleStaking is Pausable, AbstractGoodStaking, StakingToken {
 			token.transferFrom(msg.sender, address(this), _amount),
 			"transferFrom failed, make sure you approved token transfer"
 		);
-
-		FundManager fm = FundManager(nameService.getAddress("FUND_MANAGER"));
-		fm.transferInterest(address(this));
 		// approve the transfer to defi protocol
 		token.approve(address(iToken), _amount);
 		mint(_amount); //mint iToken
```

Comment:
i think we need to rename
mint and _mint is confusing

Reply:
_mintStakingToken ?